Clarify how to use Flash Attention 2 (with Pytorch Nightly installed) (#817)
Lightning-AI/lit-gpt: README.md

> **(Optional) Use Flash Attention 2 (only available in PyTorch 2.2)**

Reviewer: ```suggestion
**(Optional) Use Flash Attention 2**
```

> Flash Attention 2 will be used automatically if Pytorch 2.2 (or higher) is installed. 
> However, this is only available via the Pytorch nightly build, which you can get by running:

Reviewer: ```suggestion
Flash Attention 2 will be used automatically if PyTorch 2.2 (or higher) is installed. 
Currently, that requires installing PyTorch nightly, which you can get by running:

```
